tests/test_cli.py

```diff
@@ -327,3 +331,77 @@ def test_multiple_folders(script_runner):
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
     assert "full bbox" in ret.stdout, "joined bboxes of all files inside folder are printed to console"
+
+
+def test_zenodo_valid_link_repository(script_runner):
+    ret = script_runner.run('geoextent',
+                            '-b', '-t', 'https://zenodo.org/record/820562')
+    assert ret.success, "process should return success"
+    assert 'has no identifiable time extent' in ret.stderr
+    result = ret.stdout
+    bboxList = parse_coordinates(result)
+    assert bboxList == pytest.approx([96.21146, 25.55834, 96.35495, 25.63293], abs=tolerance)
+    assert "4326" in result
+
+
+def test_zenodo_valid_doi_repository(script_runner):
+    ret = script_runner.run('geoextent',
+                            '-b', '-t', 'https://doi.org/10.5281/zenodo.820562')
+    assert ret.success, "process should return success"
+    assert 'has no identifiable time extent' in ret.stderr
+    result = ret.stdout
+    bboxList = parse_coordinates(result)
+    assert bboxList == pytest.approx([96.21146, 25.55834, 96.35495, 25.63293], abs=tolerance)
+    assert "4326" in result
+
+
+def test_zenodo_valid_link_repository_with_no_geoextent(script_runner):
+    ret = script_runner.run('geoextent', '-b', '-t', 'https://zenodo.org/record/1810558')
+    result = ret.stdout
+    assert "bbox" not in result, "This repository contains a PDF file, it should not return a bbox"
+    assert "tbox" not in result, "This repository contains a PDF file, it should not return a tbox"
+
+
+def test_zenodo_invalid_link_repository(script_runner):
+    ret = script_runner.run('geoextent',
+                            '-b', '-t', 'https://zenado.org/record/820562')
+    assert not ret.success, 'Typo in URL'
+    assert "is not a valid" in ret.stderr, 'Typo in URL'
+
```

Comment:
Please add a test for an invalid DOI, e.g., https://doi.org/10.5281/zenodo.not.existing